src/components/Draft.js

```diff
@@ -82,7 +68,7 @@ function Page() {
           }}
           data-tabsopen={tabs.length > 0 ? '' : undefined}
           title="demo"
-          src="/demo"
+          src={`${process.env.PUBLIC_PATH || '/'}demo`}
```

Comment:
We for sure tested that this all loads correctly without a public path right? I don't recall testing it.

Reply:
Yes. It loads locally and on pages.